lib/client.ts

```diff
@@ -69,8 +72,9 @@ export class Client {
     }
   }
 
-  getAccount(): Promise<Account> {
-    return this.request(method.GET, urls.rest.account, 'account')
+  async getAccount(): Promise<Account> {
+    const rawAccount = await this.request<RawAccount>(method.GET, urls.rest.account, 'account')
+    return this.parser.parseAccount(rawAccount)
```

Comment:
Here's the general approach. I debated whether to try/catch in here. If there is an error in the API call, it will throw to the caller as it did before. If there is an error in the parsing, it will also throw to the caller, which is maybe not what we want? Would you prefer to try/catch the parsing and throw our own error rather than have the user potentially get some garbage error?

Reply:
I prefer try/catch-ing and giving the user a more specific error.

Reply:
Good call. I'll handle that in the next PR.